apps/src/applab/designElements/delete-element-button.module.scss

```diff
@@ -0,0 +1,14 @@
+@import 'color.scss';
```

Comment:
this import should use double quotes and the rules below are indented 1 tab too far. maybe i'm confused about the state of linting right now, but i thought these would've been caught when we lint?

Reply:
gah, +1
When you say "when we lint", what actions are you expecting I'm taking before pushing my changes?

Reply:
i'm expecting this to be caught before you commit because we have a pre-commit script that should run linting for the whole codebase:

https://github.com/code-dot-org/code-dot-org/blob/1ac197ffe084b3e5658d7b2b489cbeefde2b7e57/tools/hooks/pre-commit#L2

Reply:
I know what happened (I think)!
I started this PR more than a week ago, and committed changes to two components. After we finished setting up the linting rules, I merged in staging and did the other two components. Because the other commits were before the staging merge it didn't look at this file.
![image](https://user-images.githubusercontent.com/43474485/186737438-160e8cb9-736f-4e10-ab4c-81de495e34c4.png)

Reply:
I made a change to each file with problems and the linter caught them this time. What it didn't do was fix them for me. So I was alerted to the issues and then had to resolve them manually before it would allow me to commit. Progress?

Reply:
ahhh, okay, that makes sense. i think that's progress! i wonder if you could've also done something like...

```bash
git commit -am "this will fail because of linting"
# linting errors on commit
cd apps && yarn lint --fix  # or maybe you have to do `npx stylelint "**/*.scss" --fix` in order for the "--fix" option to be passed through... we can change that if so
# commit again, should pass linting
```

Reply:
Thanks - Adding that to my back pocket for next time.